README.md

```diff
@@ -9,3 +9,7 @@
 ## Requirements
 
 NDSI requires ffmpeg 3.2 to decode h.264 frames.
+
+## Windows Developer Installation
+
+Please change the `tj_dir` and `ffmpeg_libs` variables in `setup.py` as well as [`setup.py:L40`](https://github.com/pupil-labs/pyndsi/blob/302387085a3fca57cb5a9af65bf60129954a7f89/setup.py#L40) to the location of turbo-jpeg and ffmpeg installation respectively.
```

Comment:
Consider the following edit:

... to the location of turbo-jpeg and ffmpeg installations on your machine.